src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -205,20 +265,35 @@ const resolveLandingPageLinks = (
     },
   ];
 
-  const hubs = [
-    {
-      title: registered ? 'My VA health benefits' : 'VA health benefits',
-      links: myVaHealthBenefitsLinks,
-    },
-    {
-      title: 'More resources and support',
-      links: moreResourcesLinks,
-    },
-    {
-      title: 'In the spotlight',
-      links: spotlightLinks,
-    },
-  ];
+  const hubs = featureToggles[FEATURE_FLAG_NAMES.mhvMilestone2ChangesEnabled]
+    ? [
+        {
+          title: 'VA health benefits',
+          links: myVaHealthBenefitsLinks,
+        },
+        {
+          title: 'More health resources',
+          links: moreResourcesLinks,
+        },
+        {
+          title: 'In the spotlight',
+          links: spotlightLinks,
+        },
+      ]
+    : [
+        {
+          title: registered ? 'My VA health benefits' : 'VA health benefits',
```

Comment:
At some point, we stopped showing these links to unregistered folks, or when I use `USER_MOCKS.UNREGISTERED`, all I see is an alert. 

I think we want to review this file and the LandingPage.jsx as there may be an opportunity to refactor away some obsolete "unregistered" logic